contracts/OffsetHelper.sol

```diff
@@ -57,45 +56,21 @@ import "./interfaces/IToucanContractRegistry.sol";
  */
 contract OffsetHelper is OffsetHelperStorage {
     using SafeERC20 for IERC20;
-
-    /**
-     * @notice Contract constructor. Should specify arrays of ERC20 symbols and
-     * addresses that can used by the contract.
-     *
-     * @dev See `isEligible()` for a list of tokens that can be used in the
-     * contract. These can be modified after deployment by the contract owner
-     * using `setEligibleTokenAddress()` and `deleteEligibleTokenAddress()`.
-     *
-     * @param _eligibleTokenSymbols A list of token symbols.
-     * @param _eligibleTokenAddresses A list of token addresses corresponding
-     * to the provided token symbols.
-     */
-    constructor(
-        string[] memory _eligibleTokenSymbols,
-        address[] memory _eligibleTokenAddresses
-    ) {
-        uint256 i = 0;
-        uint256 eligibleTokenSymbolsLen = _eligibleTokenSymbols.length;
-        while (i < eligibleTokenSymbolsLen) {
-            eligibleTokenAddresses[
-                _eligibleTokenSymbols[i]
-            ] = _eligibleTokenAddresses[i];
-            i += 1;
-        }
-    }
+    // Chain ID of Celo mainnet
+    uint256 private constant CELO_MAINNET_CHAINID = 42220;
```

Comment:
Are we going to also store the MATIC_MAINNET_CHAINID ?

Reply:
Also we can move this to the storage contract right?

Reply:
We don't need a matic chain ID. 

As it's a constant I can't import it. But I could just make a state variable out of it. What do you think makes more sense?

Reply:
Maybe leave it as it is, we don't necessarily need to abstract this out in terms of storage layout, i think